fiftyone/core/odm/database.py

```diff
@@ -298,7 +298,7 @@ def _delete_non_persistent_datasets_if_allowed():
             )
         )
     except:
-        logger.warning(
+        logger.debug(
```

Comment:
Just noting out loud: automatic cleanup of non-persistent datasets that happens on-exit also requires read access to the `admin` database in order to function. And we've set the precedent that if we don't have that, we gracefully continue.